src/main/java/org/broadinstitute/hellbender/cmdline/programgroups/QCProgramGroup.java

```diff
@@ -0,0 +1,12 @@
+package org.broadinstitute.hellbender.cmdline.programgroups;
+
+import org.broadinstitute.barclay.argparser.CommandLineProgramGroup;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
+
+public class QCProgramGroup implements CommandLineProgramGroup {
```

Comment:
QC seems like a weird name for these.  I would either put it either in `ReadDataManipulation` or `DiagnosticsandQC`.  Alternatively, this seems like maybe a very specific tool for a certain pipeline, so maybe something like `RNAReadData` or something like that.

Reply:
Good point, I think ReadDataManipulation makes sense the most.